sound/soc/sof/ipc.c

```diff
@@ -623,7 +623,16 @@ int snd_sof_ipc_set_comp_data(struct snd_sof_ipc *ipc,
 {
 	struct snd_sof_dev *sdev = ipc->sdev;
 	struct sof_ipc_ctrl_data *cdata = scontrol->control_data;
-	int err;
+	struct sof_ipc_ctrl_data *hdr;
+	uint32_t msg_bytes = 0;
+	uint32_t hdr_bytes = 0;
+	uint32_t total_bytes = 0;
+	uint32_t send_bytes = 0;
+	uint32_t num_msg = 0;
+	uint32_t offset = 0;
```

Comment:
are all these inits required? I think not, e.g. msg_bytes, hdr_bytes, total_bytes are always set to a value.

Reply:
ok, will change